srcpkgs/PrusaSlicer/patches/cereal-is-header-only.patch

```diff
@@ -10,23 +10,24 @@ Index: PrusaSlicer-2.4.2/src/libslic3r/CMakeLists.txt
      libigl
      miniz
      boost_libs
-Index: PrusaSlicer-2.4.2/src/CMakeLists.txt
+Index: PrusaSlicer-2.5.0/src/CMakeLists.txt
 ===================================================================
---- PrusaSlicer-2.4.2.orig/src/CMakeLists.txt
-+++ PrusaSlicer-2.4.2/src/CMakeLists.txt
+--- PrusaSlicer-2.5.0.orig/src/CMakeLists.txt
++++ PrusaSlicer-2.5.0/src/CMakeLists.txt
 @@ -125,7 +125,7 @@ if (NOT WIN32 AND NOT APPLE)
      set_target_properties(PrusaSlicer PROPERTIES OUTPUT_NAME "prusa-slicer")
  endif ()
  
 -target_link_libraries(PrusaSlicer libslic3r cereal)
 +target_link_libraries(PrusaSlicer libslic3r)
+
```

Comment:
How come the patch change but the hunk indicator didn't change?

Reply:
What is that? I changed the path so that the patch applies, without it it would not apply on my side because the path is 2.5.0 and not 2.4.2.